samples/04_gis_analysts_data_scientists/address-standardization-and-correction-with-SequenceToSequence.ipynb

```diff
@@ -0,0 +1,852 @@
+{
```

Comment:
<p>For Try-it Live we avoid local data as much as possible. So, we should publish <strong>address_standardization_correction </strong>csv as <strong>csv collection </strong>and the item in prepare_textdata.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Replace&nbsp;<strong>DATA_ROOT&nbsp;</strong>with&nbsp;<strong>data_root</strong></p>